app/backplane/sensor_module/src/c_sensing_tenant.cpp

```diff
@@ -61,6 +62,13 @@ void CSensingTenant::Run() {
 #ifndef CONFIG_ARCH_POSIX
     magnetometer.UpdateSensorValue();
 #endif
+    uint32_t tmpTimestamp = 0;
+    if (int ret = rtc.GetUnixTime(tmpTimestamp); ret < 0) {
+        LOG_ERR("Failed to get time from RTC");
+    } else {
+        timestampedData.timestamp = tmpTimestamp;
```

Comment:
```c++
int CRtc::GetUnixTime(uint32_t& unixTimestamp) {
    tm time{0};
    if (int ret = GetTime(time); ret < 0) {
        return ret;
    }

    unixTimestamp = static_cast<uint32_t>(mktime(&time));
    return 0;
}
```
> On success, mktime() returns the calendar time (seconds since the Epoch), expressed as a value of type time_t.


perhaps we want timestamps at a s lightly higher frequenccy

Reply:
Done. RTC now has millis precision functions